src/dsp_tools/commands/excel2xml/excel2xml_lib.py

```diff
@@ -136,38 +137,43 @@ def find_date_in_string(string: str) -> Optional[str]:
     # sanitize input, just in case that the method was called on an empty or N/A cell
     if not check_notna(string):
         return None
+    try:
+        return _find_date_in_string_throwing(string)
+    except ValueError:
+        return None
 
-    months_dict = {
-        "January": 1,
-        "Jan": 1,
-        "February": 2,
-        "Feb": 2,
-        "March": 3,
-        "Mar": 3,
-        "April": 4,
-        "Apr": 4,
-        "May": 5,
-        "June": 6,
-        "Jun": 6,
-        "July": 7,
-        "Jul": 7,
-        "August": 8,
-        "Aug": 8,
-        "September": 9,
-        "Sept": 9,
-        "October": 10,
-        "Oct": 10,
-        "November": 11,
-        "Nov": 11,
-        "December": 12,
-        "Dec": 12,
-    }
-
-    startdate: Optional[datetime.date] = None
-    enddate: Optional[datetime.date] = None
-    startyear: Optional[int] = None
-    endyear: Optional[int] = None
 
+_months_dict = {
+    "January": 1,
+    "Jan": 1,
+    "February": 2,
+    "Feb": 2,
+    "March": 3,
+    "Mar": 3,
+    "April": 4,
+    "Apr": 4,
+    "May": 5,
+    "June": 6,
+    "Jun": 6,
+    "July": 7,
+    "Jul": 7,
+    "August": 8,
+    "Aug": 8,
+    "September": 9,
+    "Sept": 9,
+    "October": 10,
+    "Oct": 10,
+    "November": 11,
+    "Nov": 11,
+    "December": 12,
+    "Dec": 12,
+}
+
+
+def _find_date_in_string_throwing(string: str) -> str | None:
+    """
+    This method is the same as find_date_in_string(), but may raise a ValueError instead of returning None.
```

Comment:
AFAIK, this is a (standalone) function, not a (class) method (?)

```suggestion
    This function is the same as find_date_in_string(), but may raise a ValueError instead of returning None.
```

Reply:
correct... let's blame copilot :)